operations/nhc.rst

```diff
@@ -9,6 +9,12 @@ default in the ``slurmd`` charm. The base configuration file contains only
 basic checks to ensure slurm and munge processes are active. You
 can easily extend the NHC configuration to match your setup.
 
+.. note::
+
+   OSD sets the hostname of the compute nodes using only the hostname, instead
```

Comment:
```suggestion
   OSD uses a node short hostnames (`hostname -s`) as the node identifiers in the slurm. Because of this,  the NHC configuration needs to use the short hostname also. The base NHC configuration provided in the slurmd charm takes care of this by setting `* || HOSTNAME="$HOSTNAME_S"` at the top of the NHC config file. This value will not be overridden by any custom user supplied NHC config.
``` 
```